lib/msal-common/src/network/NetworkManager.ts

```diff
@@ -31,7 +32,7 @@ export class NetworkManager {
      * @param tokenEndpoint
      * @param options
      */
-    async sendPostRequest<T>(thumbprint: RequestThumbprint, tokenEndpoint: string, options: NetworkRequestOptions): Promise<NetworkResponse<T>> {
+    async sendPostRequest<T extends ServerAuthorizationTokenResponse>(thumbprint: RequestThumbprint, tokenEndpoint: string, options: NetworkRequestOptions): Promise<NetworkResponse<T>> {
```

Comment:
`T` must extend `ServerAuthorizationTokenResponse` because `ThrottlingUtils.postProcess()` expects it as an input.